Firestore/core/src/firebase/firestore/remote/serializer.h

```diff
@@ -61,11 +61,6 @@ class LocalSerializer;
 
 namespace remote {
 
-template <typename T>
```

Comment:
I moved it to `nanopb/`, partially to be able to use it in tests without depending on `remote/`, but it also seems like a more appropriate place.